more/free-programming-interactive-tutorials-en.md

```diff
@@ -339,6 +340,11 @@
 * [Selenium Tutorial - Web Automation](http://www.techbeamers.com/selenium-webdriver-tutorial)
 
 
+### Solidity
```

Comment:
:+1: Like in:

https://github.com/EbookFoundation/free-programming-books/blob/b4450227d923fcb1cd96b3b3831aecfe33d6e9b3/courses/free-courses-id.md#L42